sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -85,203 +85,267 @@ client = FormRecognizerClient(endpoint, credential)
 ## Key concepts
 
 ### FormRecognizerClient
-A `FormRecognizerClient` is the Form Recognizer interface to use for analyzing receipts and extracting layout items from 
-forms. It provides a different method based on input from a URL or input from a stream. 
-Use the receipt methods to extract receipt field values and locations from receipts from the United States from a URL
-or stream. Use the layout methods to extract table data and geometry from a URL or stream.
+A `FormRecognizerClient` is the Form Recognizer interface to use for recognizing data from forms using custom trained models,
+recognizing content and layout from forms, and analyzing receipts. It provides different methods
+based on inputs from a URL and inputs from a stream.
 
-### CustomFormClient
-A `CustomFormClient` is the Form Recognizer interface to use for creating, using, and managing custom machine-learned models. 
-It provides operations for training models on forms you provide, and extracting field values and locations from your 
-custom forms.  It also provides operations for viewing and deleting models, as well as understanding how close you 
-are to reaching subscription limits for the number of models you can train.
+#### Recognizing Custom Forms
+Our endpoint for recognizing custom forms takes in the ID of a custom model. The model can be trained either using this [library](#training-models "Training models")
+or via a UI such as the service's [labelling tool][fr-labelling-tool].
+Be sure to note that the form type you are recognizing should be the included in the types of forms used to train this custom model.
```

Comment:
This feels like a detail that would go better in the training section.

Reply:
Ok I'll remove it. I originally had it here because I really wanted to stress you only recognize custom forms of form type that your custom model was trained on, but I have it elsewhere too. Thanks!